docs/quick_start/starting.rst

```diff
@@ -37,4 +37,16 @@ In both cases the user will be prompted to enter a password. It is possible, but
 
    The *start*, *status* and *help* commands do not require authentication.
 
+It is also possible to use certificate based authentication to login to the system. In this case the "fledge" command line utility should be passed the *-c* flag with the name of the certificate file to use to authenticate.
+
+.. code-block:: console
+
+   $ /usr/local/fledge/bin/fledge -c ~/.fledge/admin.cert stop
+   Stopping Fledge..........
+   Fledge Stopped
+
+.. note::
+
+   Extreme caution should be taken when storing certificate files that they not be readable by any other user within the system.
```

Comment:
This note is intended for Admins who are responsible for setting up the FL instance with user-specific configurations. Only they have the ability to stop the FL instance using a certificate.
Given these conditions, we can proceed forward.Although the `systemctl stop` issue persists, a separate ticket to address it.